### PR TITLE
Custom Pagination bar functionality

### LINKED ISSUE #20: implement Movielist Pagination bar
**Is your feature request related to a problem? Please describe.**
Movielist needs a way to display all movie data from api, instead of just 10 movie results.

**Describe the solution you'd like**
Using a pagination bar at the bottom, user can click on different pages to view more movie data from api

### PR DESCRIPTION
**Pull request by dannysanchez559**
## Changes
1. Implement Pagination component
2. Refactor api call, and functions for Pagination functionality
3. Basic Pagination styling

## Purpose
Needed a way to return more movie data from api, api only returns 10 moviecards unless there is a specific page number which shows more moviecard content.

## Approach
Pagination allows user to click on a page that displays additional movie data.

Closes #20

### CODE REVIEW

File: movie-app/src/components/Header.js [hunk 1]
```diff
@@ -1,32 +1,24 @@
...
-  const [movieName, setMovieName] = useState("");
 
-  const onSubmit = (e) => {
-    e.preventDefault();
+  const onSubmit = () => {
 
-    if (movieName.length) {
-      props.getMovieSearchResultsApi(movieName);
+    if (props.movieName.length) {
+      props.getMovieSearchResultsApi(props.movieName);
```
Comment (Lap343) [inline movie-app/src/components/Header.js:8]: nice checking to see if they had something in movie name. Don't know if it was the same for you but Our API call wouldn't even come back if it was less than 3 letters. No changes needed just saying though.

**Review by Lap343 — APPROVED** [review]
Paginator section looks good for a merge.

File: movie-app/src/components/PaginationBar.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
+import React, { useState } from 'react';
```
Comment (krferrell) [inline movie-app/src/components/PaginationBar.js:1]: Nice job with the pagination! Looks great.

**Review by krferrell — APPROVED** [review]
Well done. I like seeing all the checking before rendering content. The pagination looks good too. Keep up the great work team!

**Review by connielion — COMMENTED** [review]
Very nice. Keep up the good work.

File: movie-app/src/App.js [hunk 3]
```diff
@@ -4,14 +4,22 @@ import { getTopNetflixMovies, getMovieSearchResults } from "./utils/api";
 import Header from "./components/Header";
 import Carousel from "./components/Carousel";
 import MovieList from "./components/MovieList";
+import PaginationBar from "./components/PaginationBar";
 
 const App = () => {
   const [netflixMovies, setNetflixMovies] = useState([]);
   const [movieList, setMovieList] = useState([]);
+  const [totalResults, setTotalResults] = useState(0);
+  const [movieName, setMovieName] = useState("");
```
Comment (gonzalespaulb) [inline movie-app/src/App.js:13]: Great job adding states on the top level. Makes for better readability

File: movie-app/src/components/Header.js [hunk 4]
```diff
@@ -1,32 +1,24 @@
...
   return (
     <div className="header">
       <h1>Movie App Title</h1>
 
-      <form onSubmit={onSubmit}>
-        <input type="text" placeholder="Enter movie name" name="searchbar" value={movieName} onChange={onChange}/>
-        <button id="searchbutton" type="submit"></button>
-      </form>
+        <input type="text" placeholder="Enter movie name" name="searchbar" value={props.movieName} onChange={props.onMovieNameChange}/>
+        <button id="searchbutton" onClick={() => onSubmit()}></button>
```
Comment (gonzalespaulb) [inline movie-app/src/components/Header.js:20]: Change the function that is being called onSubmit to another name. This might cause bugs in the future.

**Review by gonzalespaulb — COMMENTED** [review]
Great job on the pagination functionality. Just needs a couple function name changes to prevent future bugs.

**Pull request merged**